dev/com.ibm.ws.cdi.1.2_fat/.gitignore

```diff
@@ -0,0 +1,2 @@
+/test-applications-overlay
+/publish\bundles
```

Comment:
should entries in the .gitignore be absolute or relative ... should these start with a `/` or not?

Reply:
I checked a couple random ones. They both started with /